docs/api-specs/01_auth.md

```
@@ -0,0 +1,340 @@
+| `isNewUser` | Boolean | `true`: 신규 가입 (닉네임 설정 화면으로 이동), `false`: 기존 회원 |
+
+### Error
+
+| Status | code | 상황 |
+|--------|------|------|
+| 400 | `INVALID_ID_TOKEN` | Firebase ID Token 검증 실패 |
+| 400 | `UNSUPPORTED_PLATFORM` | socialPlatform이 GOOGLE/APPLE이 아닌 경우 |
+
+### 서버 처리 로직
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)